dags/analysis.py

```diff
@@ -0,0 +1,25 @@
+from airflow import DAG
+from airflow.operators.sensors import ExternalTaskSensor
+from datetime import datetime, timedelta
+from utils.gcp import bigquery_etl_query
+
+default_args = {
+    "depends_on_past": False,
+    "start_date": datetime(2020, 3, 29),
+    "owner": "ascholtz@mozilla.com",
+    "email": [
+        "telemetry-alerts@mozilla.com",
+        "jmccrosky@mozilla.com",
+        "ascholtz@mozilla.com",
+    ],
+    "email_on_failure": True,
+    "retries": 2,
+    "retry_delay": timedelta(minutes=30),
+}
+
+with DAG("public_analysis", default_args=default_args, schedule_interval="0 6 * * *") as dag:
```

Comment:
The databricks notebook gets triggered at 3am UTC and runs 1-2 hours (if I understood correctly). Running this at 6am should be safe enough?